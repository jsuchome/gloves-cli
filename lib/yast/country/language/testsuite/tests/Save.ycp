/**
 * tests for Language::Save
 * Author:	jsuchome@suse.cz
 * $Id: Save.ycp 32769 2006-09-08 09:29:15Z jsuchome $
 */
{

    // testedfiles: Language.ycp

    include "testsuite.ycp";

    map READ = $[
	"sysconfig" : $[
	    "language" : $[
		"RC_LANG"		: "en_US.UTF-8",
		"RC_LC_MESSAGES"	: "",
	    ],
	],
	"target" : $[
	    "bash_output"	: $[],
	    "size"		: 1,
	    "yast2"		: $[],
	    "dir"		: [],
	],
    ];

    TESTSUITE_INIT([READ,$[],READ], nil);

    import "Language";

    Language::languages	= "en_US";

    TEST(``(Language::Save ()), [READ, $[], $[]], nil);
}
