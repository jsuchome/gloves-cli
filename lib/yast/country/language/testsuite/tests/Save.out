Read	.target.size "/etc/sysconfig/language" 1
Read	.target.dir "/usr/share/YaST2/data/languages" []
Write	.sysconfig.language.RC_LANG "en_US.UTF-8" true
Read	.sysconfig.language.RC_LC_MESSAGES ""
Write	.sysconfig.language.ROOT_USES_LANG "ctype" true
Write	.sysconfig.language.INSTALLED_LANGUAGES "en_US" true
Write	.sysconfig.language nil true
Return	nil
